items/equip-2hand/item5261_Launcher.xml

```diff
@@ -6,7 +6,7 @@
     <its:translateRule selector="//item/@effect" translate="yes"/>
     <its:translateRule selector="//*" translate="no"/>
   </its:rules>
-  <item id="5261" image="equipment/weapon/xmas-slingshot-icon.png" name="Candy-Caned Anti-Grinch-Rifle 2020" description="The best way to shoot snowballs." type="equip-2hand" attack-action="attack_bow" attack="5" attack-range="3" weight="10" critical-hit-effect-id="55" missile-particle="graphics/particles/snowball.particle.xml">
+  <item id="5261" image="equipment/weapon/xmas-slingshot-icon.png" name="Candy-Caned Anti-Grinch-Rifle 2020" description="The best way to shoot snowballs." type="equip-2hand" attack-action="attack_bow" attack="20" attack-range="3" weight="10" critical-hit-effect-id="55" missile-particle="graphics/particles/snowball.particle.xml">
```

Comment:
My bad, I meant "equip-1hand" but wrote "equip-2hand" instead

I'm 99.99% sure you can equip the snow launcher with a shield (if you fix here)

Hey, I wanted it to be extra cool B)